### DISCUSSION TITLE
Does the overall update script supposed to also update individual software in containers?

### DISCUSSION THREAD

**Discussion by gezuka77** [tteck/Proxmox#3939] [Help Needed] [answered]
Hi,

Does this  `bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/update-lxcs.sh)"` script also expected to update software in the containers or just the container OS?
I don't think so, but if not how to do "global" update. I now have to do this one-by-one where an update script is available.

**Comment by tteck** [comment, marked as answer]
Only the container OS. There is no global application update.

**Reply by gezuka77** [reply]
Thanks!

**Comment by Mellowlynx** [comment]
I would recommend deploying Ansible on a LXC and making a playbook to update everything.
Takes some time, also when you move a LXC to another node, you need to update the playbook, but works great overall.

**Reply by gezuka77** [reply]
Thanks!

**Reply by di5cord20** [reply]
I've never used Ansible but am curious to learn.  Any suggestions for tutorials or quick start guides to start experimenting?  I like the use case you mentioned.

**Reply by Mellowlynx** [reply]
I'm learning it through company information, so can't share that.
But a good start after installing is here: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_intro.html
Also, on YouTube you can find allot.

To make it more fun for me, I also have a local Github running, where the playbook is pulled from. But you can also use Gethub.com.
For running tasks, there are multiple options to run through next to the CLI.
Rundeck is a great one, but [Semaphore](https://semaphoreui.com/) is also good for home lab use.
